src/OpenTelemetry.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -164,5 +170,35 @@ internal static Settings FromDefaultSources()
 
             return new Settings(configurationSource);
         }
+
+        private static OtlpExportProtocol? GetExporterOtlpProtocol(IConfigurationSource source)
+        {
+            // the defualt in SDK is grpc. http/protobuf should be default for our purposes
+            var exporterOtlpProtocol = source.GetString(ConfigurationKeys.ExporterOtlpProtocol);
+
+            if (string.IsNullOrEmpty(exporterOtlpProtocol) || exporterOtlpProtocol == "http/protobuf")
+            {
+                // override settings only for http/protobuf
+                // the second part of the condition is needed to override default endpoint as long as https://github.com/open-telemetry/opentelemetry-dotnet/pull/2868 is not fixed
+                return OpenTelemetry.Exporter.OtlpExportProtocol.HttpProtobuf;
+            }
+
+            // null value here means that it will be handled by OTEL .NET SDK
+            return null;
+        }
+
+        private static Uri GetExporterOtlpEndpoint(IConfigurationSource source, OtlpExportProtocol? otlpExportProtocol)
+        {
+            var exporterOtlpEndpoint = source.GetString(ConfigurationKeys.ExporterOtlpEndpoint);
+
+            if (!otlpExportProtocol.HasValue || otlpExportProtocol.Value != OpenTelemetry.Exporter.OtlpExportProtocol.HttpProtobuf || !string.IsNullOrWhiteSpace(exporterOtlpEndpoint))
+            {
+                // null value here means that it will be handled by OTEL .NET SDK
+                return null;
+            }
+
+            // override endpoint only for otlp over http/protobuf
+            return new Uri("http://localhost:4318/v1/traces");
```

Comment:
how about the following in order to have the flow similar to the one in `GetExporterOtlpProtocol`?

```suggestion
            if (otlpExportProtocol == OpenTelemetry.Exporter.OtlpExportProtocol.HttpProtobuf && string.IsNullOrWhiteSpace(exporterOtlpEndpoint))
            {
                // override endpoint only for otlp over http/protobuf
                return new Uri("http://localhost:4318/v1/traces");
            }

            // null value here means that it will be handled by OTEL .NET SDK
            return null;
```

also I think the condition is simpler a little bit